src/widgets/settingspages/GeneralPageView.hpp

```diff
@@ -99,15 +99,17 @@ class GeneralPageView : public QWidget
     SubtitleLabel *addSubtitle(const QString &text);
     /// @param inverse Inverses true to false and vice versa
     QCheckBox *addCheckbox(const QString &text, BoolSetting &setting,
-                           bool inverse = false);
-    ComboBox *addDropdown(const QString &text, const QStringList &items);
+                           bool inverse = false, QString toolTipText = nullptr);
```

Comment:
While this isn't a blocker, I dislike the usage of `nullptr` here. To me the `nullptr` literal means one thing: a pointer with the value 0 and QString isn't a pointer per se.